### DISCUSSION TITLE
Interactive login browser window (login.microsoftonline.com) will not open after publishing it in IIS

### DISCUSSION THREAD

**Discussion by georgesndavid** [pnp/pnpcore#1489] [Q&A]
We are using PNP Core SDK (PNP.Core.Auth) to make connection between .NET Core Mvc web application and SharePoint online. Everything works perfectly fine at local with an interactive login browser window (login.microsoftonline.com). The day after we deployed the project under IIS window hosting machine the interactive login browser window no longer opening up for authentication. We have spotted below error in the event viewer system window.

**The application-specific permission settings do not grant Local Activation permission for the COM Server application with CLSID {3480A401-BDE9-4407-BC02-798A866AC051} and APPID {30AD8C8E-AE85-42FA-B9E8-7E99E3DFBFC5} to the user IIS APPPOOL\XXXXX SID (S-1-5-82-2767913229-2882314508-2470621768-1801221961-134645414) from address LocalHost (Using LRPC) running in the application container Unavailable SID (Unavailable). This security permission can be modified using the Component Services administrative tool.**

Could anyone please check and clarify above error if it is related to the PNP Core SDK. Does DCOM permissions needs to be handled in order make PNP Core SDK work properly?

Thanks in advance

**Comment by jansenbe** [comment]
There's no dependency on DCOM in PnP Core SDK...no idea what might be causing this.

**Reply by georgesndavid** [reply]
@jansenbe thank you for confirming this. May be an permission issue which I need to take care of it internally. I will do my further research and update here. 

George
